### ISSUE TITLE
variable PLAY_SERVICES_VERSION not honored with cordova 8.1.2 and cordova android 6.4

### ISSUE THREAD

**Issue by DavidWiesner** [dpa99c/cordova-android-play-services-gradle-release#17]
**I'm submitting a ...**  (check one with "x"):
- [x] bug report
- [ ] feature request
- [ ] documentation issue

# Bug report

**Current behavior:**
- PLAY_SERVICES_VERSION not honored when building project with cordova 8.1.2 and cordova android 6.4.0
- in the build process it is still trying to install the version mentioned in the plugin or latest version of that 
e.g.: cordova-plugin-request-location-accuracy want to install version 15.0.1 and I overwrite this with your plugin to '15.0.0'. But `cordova build` will still trying to fetch the original version see this error message:
```
> Could not find play-services-basement.aar (com.google.android.gms:play-services-basement:15.0.1).
  Searched in the following locations:
      https://jcenter.bintray.com/com/google/android/gms/play-services-basement/15.0.1/play-services-basement-15.0.1.aar
``` 
- btw.: all other combinations: cordova 6.5.0 + cordova-android 6.4.0 and cordova 8.x + cordova-android 7.x work like expected

**Expected behavior:**

- `cordova build`should install the version specified by the variable PLAY_SERVICES_VERSION

**Steps to reproduce:**
- install cordova 8.1.2 or 8.0.0 
- create a new project 
- go to directory of the new project
- install a plugin that requires play services e.g.: cordova-plugin-request-location-accuracy
- install your plugin with variable PLAY_SERVICES_VERSION setted 
- add cordova android platform with version 6.4.0
- run cordova build
here as bash commands
```
npm install -g cordova@8.1.2
cordova create cordova-8-android-6
cd cordova-8-android-6
cordova plugin add cordova-plugin-request-location-accuracy
cordova plugin add cordova-android-play-services-gradle-release --variable PLAY_SERVICES_VERSION=15.0.0
cordova platform add android@6.4.0
cordova build android
``` 

**Environment information**

- Cordova CLI version 
8.1.2 (cordova-lib@8.1.1)
- Cordova platform version
Installed platforms:
  android 6.4.0
- Plugins & versions installed in project (including this plugin)
cordova-android-play-services-gradle-release 1.4.6 "cordova-android-play-services-gradle-release"
cordova-plugin-request-location-accuracy 2.2.3 "Request Location Accuracy"
cordova-plugin-whitelist 1.3.3 "Whitelist"
- Dev machine OS and version, e.g.
ProductName:	Mac OS X
ProductVersion:	10.13.6
BuildVersion:	17G65
	
_Android build issue:_	
- Node JS version
v8.9.4
- Gradle version
4.1			4.3.1			buildOutputCleanup
- Target Android SDK version
26
- Android SDK details
	- `sdkmanager --list | sed -e '/Available Packages/q'`

**Related code:**

```
npm install -g cordova@8.1.2
cordova create cordova-8-android-6
cd cordova-8-android-6
cordova plugin add cordova-plugin-request-location-accuracy
cordova plugin add cordova-android-play-services-gradle-release --variable PLAY_SERVICES_VERSION=15.0.0
cordova platform add android@6.4.0
cordova build android
```

**Console output**

<details>
<summary>console output</summary>

```

Adding cordova-plugin-request-location-accuracy to package.json
Saved plugin info for "cordova-plugin-request-location-accuracy" to config.xml
Adding cordova-android-play-services-gradle-release to package.json
Saved plugin info for "cordova-android-play-services-gradle-release" to config.xml
Using cordova-fetch for cordova-android@6.4.0
Adding android project...
Creating Cordova project for the Android platform:
	Path: platforms/android
	Package: io.cordova.hellocordova
	Name: HelloCordova
	Activity: MainActivity
	Android target: android-26
Subproject Path: CordovaLib
Android project created with cordova-android@6.4.0
Discovered plugin "cordova-plugin-whitelist" in config.xml. Adding it to the project
Installing "cordova-plugin-whitelist" for android

               This plugin is only applicable for versions of cordova-android greater than 4.0. If you have a previous platform version, you do *not* need this plugin since the whitelist will be built in.
          
Adding cordova-plugin-whitelist to package.json
Saved plugin info for "cordova-plugin-whitelist" to config.xml
Discovered plugin "cordova-plugin-request-location-accuracy" in config.xml. Adding it to the project
Installing "cordova-plugin-request-location-accuracy" for android
Subproject Path: CordovaLib
Adding cordova-plugin-request-location-accuracy to package.json
Saved plugin info for "cordova-plugin-request-location-accuracy" to config.xml
Discovered plugin "cordova-android-play-services-gradle-release" in config.xml. Adding it to the project
Installing "cordova-android-play-services-gradle-release" for android
cordova-android-play-services-gradle-release: Android platform: V6
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/build.gradle
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/cordova-android-play-services-gradle-release/properties.gradle
Adding cordova-android-play-services-gradle-release to package.json
Saved plugin info for "cordova-android-play-services-gradle-release" to config.xml
cordova-android-play-services-gradle-release: Android platform: V6
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/build.gradle
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/cordova-android-play-services-gradle-release/properties.gradle
--save flag or autosave detected
Saving android@~6.4.0 into config.xml file ...
cordova-android-play-services-gradle-release: Android platform: V6
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/build.gradle
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/cordova-android-play-services-gradle-release/properties.gradle
cordova-android-play-services-gradle-release: Android platform: V6
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/build.gradle
cordova-android-play-services-gradle-release: Wrote custom version '15.0.0' to /Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/cordova-android-play-services-gradle-release/properties.gradle
ANDROID_HOME=/Users/dboho/Library/Android/sdk
JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_121.jdk/Contents/Home
Parallel execution is an incubating feature.
:wrapper

BUILD SUCCESSFUL in 0s
1 actionable task: 1 executed
Subproject Path: CordovaLib
Parallel execution is an incubating feature.
The Task.leftShift(Closure) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use Task.doLast(Action) instead.
	at build_759f271iwkml710t2aaju8enm.run(/Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/build.gradle:141)
Configuration 'compile' in project ':' is deprecated. Use 'implementation' instead.
publishNonDefault is deprecated and has no effect anymore. All variants are now published.
:preBuild UP-TO-DATE
:CordovaLib:preBuild UP-TO-DATE
:CordovaLib:preDebugBuild UP-TO-DATE
:CordovaLib:checkDebugManifest
:prepareLintJar
:generateDebugResValues
:createDebugCompatibleScreenManifests
:splitsDiscoveryTaskDebug
:mergeDebugShaders
:compileDebugShaders
:generateDebugAssets
:mergeDebugJniLibFolders
:CordovaLib:processDebugManifest
:validateSigningDebug
:preDebugBuild
:CordovaLib:compileDebugAidl
:CordovaLib:packageDebugRenderscript NO-SOURCE
:CordovaLib:compileDebugRenderscript
:CordovaLib:generateDebugResValues
:CordovaLib:generateDebugResources
:CordovaLib:packageDebugResources
:CordovaLib:platformAttrExtractor
:CordovaLib:generateDebugBuildConfig
:preDebugBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':debugCompileClasspath'.
> Could not find play-services-basement.aar (com.google.android.gms:play-services-basement:15.0.1).
  Searched in the following locations:
      https://jcenter.bintray.com/com/google/android/gms/play-services-basement/15.0.1/play-services-basement-15.0.1.aar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
17 actionable tasks: 17 executed
/Users/dboho/experiments/cordova-example/cordova-8-1-2/platforms/android/gradlew: Command failed with exit code 1 Error output:
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':debugCompileClasspath'.
> Could not find play-services-basement.aar (com.google.android.gms:play-services-basement:15.0.1).
  Searched in the following locations:
      https://jcenter.bintray.com/com/google/android/gms/play-services-basement/15.0.1/play-services-basement-15.0.1.aar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s

```

</details><br/><br/>

**Comment by dpa99c** [comment]
This is a side-effect of my attempts to address the backwardly-incompatible changes in `cordova-android@7.1.2`. Please use v1.4.4 of this plugin until I can fix it for both `cordova-android@6` and `cordova-android@7.1.2`

**Comment by DavidWiesner** [comment]
@dpa99c i found this commit 5ac6f48bc where you removed the framework path is this related to the problem?

**Comment by dpa99c** [comment]
Yep, was related to that. I've reworked the plugin to simplify it and use a single Gradle file and published to npm as v2.0.0

**Comment by DavidWiesner** [comment]
This was an amazing fast bug fix. Thank you!
